docs/index.md

```diff
@@ -14,12 +14,13 @@
 - No guesswork on finding the right version of a component – they all undergo thorough testing to ensure compatibility
 - Freely available reference architectures for solutions like high-availability, backups and disaster recovery 
 - Spatial data handling support via PostGIS
+- Transparent Data Encryption via `pg_tde`.
 - Monitoring of the database health, performance and infrastructure usage via open source [Percona Management and Monitoring :octicons-link-external-16:](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) with PostgreSQL-specific dashboards
 - Run PostgreSQL on Kubernetes using open source [Percona Operator for PostgreSQL :octicons-link-external-16:](https://docs.percona.com/percona-operator-for-postgresql/2.0/index.html). It not only automates deployment and management of PostgreSQL clusters on Kubernetes, but also includes enterprise-ready features for high-availability, backup and restore, replication, logging, and more 
```

Comment:
Could we add an Everest bullet here the same way we have one for the Operator?

Reply:
Added